local-compose.yml

```diff
@@ -0,0 +1,33 @@
+nginx:
+    extends:
+        file: docker-compose.yml
+        service: nginx
+    build: .
+    mem_limit: 128g
+    links:
+        - consul:consul
+    environment:
+        - CONSUL=consul
```

Comment:
Duplicate `links` stanza removed, but we still need to set the environment var for it, though.